sdk/playwrighttesting/microsoft-playwright-testing/src/utils/reporterUtils.ts

```diff
@@ -318,8 +318,10 @@ class ReporterUtils {
   };
 
   public static isTimeGreaterThanCurrentPlus10Minutes(isoString: string): boolean {
+    // Convert UTC to local
+    const localTimeString = new Date(isoString).toLocaleString();
     // Parse ISO 8601 string into a timestamp
-    const timestampFromIsoString: number = Date.parse(isoString);
+    const timestampFromIsoString: number = Date.parse(localTimeString);
```

Comment:
This will break if it is not in the date time format (can be different for different locales) - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse